build.gradle

```diff
@@ -22,17 +38,13 @@ dependencies {
 
 	// Fabric API. This is technically optional, but you probably want it anyway.
 	modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
-
+	
 	// Uncomment the following line to enable the deprecated Fabric API modules. 
 	// These are included in the Fabric API production distribution and allow you to update your mod to the latest modules at a later more convenient time.
 
 	// modImplementation "net.fabricmc.fabric-api:fabric-api-deprecated:${project.fabric_version}"
 }
 
-base {
-	archivesName = project.archives_base_name
-}
```

Comment:
This needs adding back, will need fixing in the template gen first.

Reply:
Ugh but it got moved up in the buildscript?

Reply:
This is above, doesnt need fixing.